### DISCUSSION TITLE
Edit AndroidManifest manually

### DISCUSSION THREAD

**Discussion by odoookentang** [Sketchware-Pro/Sketchware-Pro#751] [Q&A] [answered]
Since i'm trying to make an app that locks the user from using another app, i need to restrict the users movement. So far this has been easy, but since the ability to edit AndroidManifest is nonexistent, i xan't do this anymore.

**Comment by j-px** [comment, marked as answer]
You can and should use XML Command Blocks for that. A tutorial on them is [available in our Discord server](https://discord.com/channels/790686719753846785/797801500276424724/798164544375619625).
